README.md

```diff
@@ -91,9 +91,8 @@ We're providing a Docker image to build the documentation.
 
```

Comment:
Please, add info about native compilation, because this action might be very tricky. I propose the following command for that purpose `TEXINPUTS=./scn: latexmk -pdf -bibtex main.tex`